roentgen/_astropy_init.py

```diff
@@ -1,4 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+# probably don't need this anymore!
```

Comment:
Yeah if this is still in the base template, we should remove it.

Reply:
I did not use the new template. I was trying to migrate myself.

Reply:
Oh in that case, we can remove this but will need some code in setup.py and in another file.

Reply:
What do you mean given that this file is not used at all and everything seems to work now. What code would need to be added in other places?

Reply:
I think it should be imported by the ``__init__.py`` from what i recall from the old template.